docs/occ.md

```diff
@@ -19,7 +19,7 @@ Install a new bot on the server
 |---|---|---|---|---|---|
 | `--output` | Output format (plain, json or json_pretty, default is plain) | yes | no | no | `'plain'` |
 | `--no-setup` | Prevent moderators from setting up the bot in a conversation | no | no | no | `false` |
-| `--feature\|-f` | Specify the list of features for the bot - webhook: The bot receives posted chat messages as webhooks - response: The bot can post messages and reactions as a response - none: When all features should be disabled for the bot | yes | yes | yes | *Required* |
+| `--feature\|-f` | Specify the list of features for the bot - webhook: The bot receives posted chat messages as webhooks - response: The bot can post messages and reactions as a response - event: The bot reads posted messages from local events - none: When all features should be disabled for the bot | yes | yes | yes | *Required* |
```

Comment:
Does it also do something with the events? Maybe that can be mentioned in the docs?
```suggestion
| `--feature\|-f` | Specify the list of features for the bot - webhook: The bot receives posted chat messages as webhooks - response: The bot can post messages and reactions as a response - event: The bot reads posted messages from local events and acts on the predefined triggers (if any) - none: When all features should be disabled for the bot | yes | yes | yes | *Required* |
```

Something like that maybe?